### PR TITLE
[FIX] Remove sidebar header on admin embedded version

### LINKED ISSUE #8333: Support embedded layout on /admin URL for mobile apps
We need a way to open the admin options in a web view with **no** options to come back to a conversation.

Today, if we call `?layout=embedded` on admin URL, it doesn't change anything.

### PR DESCRIPTION
**Pull request by karlprieb**
@RocketChat/core 

Closes #8333 

screenshot-2017-9-27 1 rocket chat

### CODE REVIEW

File: packages/rocketchat-ui/client/views/app/burger.js [hunk 1]
```diff
@@ -6,5 +6,9 @@ Template.burger.helpers({
 		if (Session.equals('isMenuOpen', true)) {
 			return 'menu-opened';
 		}
+	},
+	embeddedVersion() {
+		console.log('burger', RocketChat.Layout.isEmbedded());
```
Comment (felipevolpatto) [inline packages/rocketchat-ui/client/views/app/burger.js]: Ops!

Comment (karlprieb) [reply packages/rocketchat-ui/client/views/app/burger.js]: Sorry :(

File: packages/rocketchat-ui-account/client/accountFlex.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-			{{> accountBox lighten=true modifier="--account" }}
-			<button class="sidebar-flex__back-button" data-action="back">
-				{{> icon block="sidebar-flex__back-icon" icon="back"}}
-				<span>{{_ "Back_to_chat"}}</span>
-			</button>
-		</header>
-		<div class="rooms-list">
+		{{#unless embeddedVersion}}
+			<header class="sidebar__header">
+				{{> accountBox lighten=true modifier="--account" }}
```
Comment (engelgabriel) [inline packages/rocketchat-ui-account/client/accountFlex.html]: Remove this account box

File: packages/rocketchat-ui-admin/client/adminFlex.html [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-			{{> accountBox lighten=true modifier="--admin"}}
-			<button class="sidebar-flex__back-button" data-action="back">
-				{{> icon block="sidebar-flex__back-icon" icon="back"}}
-				<span>{{_ "Back_to_chat"}}</span>
-			</button>
-		</header>
-		<div class="rooms-list" aria-label="{{_ "Administration"}}">
+		{{#unless embeddedVersion}}
+			<header class="sidebar__header">
+				{{> accountBox lighten=true modifier="--admin"}}
```
Comment (engelgabriel) [inline packages/rocketchat-ui-admin/client/adminFlex.html]: Also remove this account box

File: packages/rocketchat-ui-admin/client/adminFlex.html [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+				<button class="sidebar-flex__back-button" data-action="back">
+					{{> icon block="sidebar-flex__back-icon" icon="back"}}
+					<span>{{_ "Back_to_chat"}}</span>
```
Comment (engelgabriel) [inline packages/rocketchat-ui-admin/client/adminFlex.html]: Change "Back_to_chat" to just "Back"

File: packages/rocketchat-ui-account/client/accountFlex.html [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
+				<button class="sidebar-flex__back-button" data-action="back">
+					{{> icon block="sidebar-flex__back-icon" icon="back"}}
+					<span>{{_ "Back_to_chat"}}</span>
```
Comment (engelgabriel) [inline packages/rocketchat-ui-account/client/accountFlex.html]: Change "Back_to_chat" to just "Back"

File: packages/rocketchat-ui-account/client/accountFlex.html (hunk 2 above)
Comment (karlprieb) [reply packages/rocketchat-ui-account/client/accountFlex.html]: I can remove that, but I won't do that on this PR. This PR is about embedded view and not about accountBox.

File: packages/rocketchat-ui-admin/client/adminFlex.html (hunk 3 above)
Comment (karlprieb) [reply packages/rocketchat-ui-admin/client/adminFlex.html]: I can remove that, but I won't do that on this PR. This PR is about embedded view and not about accountBox.

File: packages/rocketchat-ui-account/client/accountFlex.html (hunk 5 above)
Comment (karlprieb) [reply packages/rocketchat-ui-account/client/accountFlex.html]: Why?

File: packages/rocketchat-ui-admin/client/adminFlex.html (hunk 4 above)
Comment (karlprieb) [reply packages/rocketchat-ui-admin/client/adminFlex.html]: Why?

Comment (engelgabriel) [reply packages/rocketchat-ui-admin/client/adminFlex.html]: Less is More

Comment (karlprieb) [reply packages/rocketchat-ui-admin/client/adminFlex.html]: ok, it's easy to maintain translations :)

**Pull request merged**